external/upstream/cppe/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+if(${ENABLE_cppe})
+    find_package(cppe 0.0.0 CONFIG QUIET)
+
+    if(${cppe_FOUND})
+        get_property(_loc TARGET cppe::cppe PROPERTY LOCATION)
+        message(STATUS "${Cyan}Found cppe${ColourReset}: ${_loc} (found version ${cppe_VERSION})")
+        add_library(cppe_external INTERFACE)  # dummy
+    else()
+        if(${CMAKE_INSIST_FIND_PACKAGE_cppe})
+            message(FATAL_ERROR "Suitable cppe could not be externally located as user insists")
+        endif()
+
+        include(ExternalProject)
+        message(STATUS "Suitable cppe could not be located, ${Magenta}Building cppe${ColourReset} instead.")
+        ExternalProject_Add(cppe_external
+            GIT_REPOSITORY https://github.com/maxscheurer/cppe
+            GIT_TAG master
+            # UPDATE_COMMAND ""
+            CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${STAGED_INSTALL_PREFIX}
+                       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+                       -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+                       -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+                       -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
+                       -DCMAKE_INSTALL_BINDIR=${CMAKE_INSTALL_BINDIR}
+                       -DCMAKE_INSTALL_DATADIR=${CMAKE_INSTALL_DATADIR}
+                       -DCMAKE_INSTALL_INCLUDEDIR=${CMAKE_INSTALL_INCLUDEDIR}
+                       -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
+                       -DENABLE_OPENMP=${ENABLE_OPENMP}
+                       -DENABLE_PYTHON_INTERFFACE=OFF
```

Comment:
I think you have a typo in this option.

Reply:
Corrected in e8ab7d0b0e153ff326918521bd26cc13d738b481.